docs/scheduler/configuration/logging.rst

```diff
@@ -1,4 +1,4 @@
-.. _backend-configuration-logging:
+.. _server-configuration-logging:
```

Comment:
This file is in `docs/scheduler` folder, not `docs/server`. Probably current prefix `backend-configuration` is okay, as this is used in all backend components (server, scheduler, worker)